Tests/fixtures/tv-1.yml

```diff
@@ -4092,23 +4092,1200 @@
+  os:
+    name: Android
+    version: "10"
+    platform: ""
+  client:
+    type: browser
+    name: Opera Devices
+    version: 4.21.0.273
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: Sony
+    model: BRAVIA-VH1
+  os_family: Android
+  browser_family: Opera
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.1.1; ATMACA_SmartTV) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.185 Mobile Safari/537.36
+  os:
+    name: Android
+    version: 7.1.1
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 86.0.4240.185
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: Atmaca Elektronik
+    model: ""
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.1.2; EVBOX-PLUS+) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.104 Safari/537.36
+  os:
+    name: Android
+    version: 7.1.2
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 96.0.4664.104
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: EVPAD
+    model: EVBox Plus
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.1.2; Kingbox) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.91 Safari/537.36
+  os:
+    name: Android
+    version: 7.1.2
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 90.0.4430.91
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: Kingbox
+    model: ""
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 4.4.2; MBox3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
+  os:
+    name: Android
+    version: 4.4.2
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 81.0.4044.138
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: MBOX
+    model: ""
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Andr0id 4.4.2; BHX-UH200 Build/KOT49H) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.82 Safari/537.36 OPR/32.0.2070.0 OMI/4.7.0.34.release
+  os:
+    name: Android
+    version: 4.4.2
+    platform: ""
+  client:
+    type: browser
+    name: Opera Devices
+    version: 4.7.0.34
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: SK Broadband
+    model: BHX-UH200
+  os_family: Android
+  browser_family: Opera
+-
+  user_agent: Mozilla/5.0 (Linux; QM152E Build/LMY47D) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.125 Safari/537.36 OPR/31.0.1890.0 OMI/4.6.1.13.Sprinter.13
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: ""
+  client:
+    type: browser
+    name: Opera Devices
+    version: 4.6.1.13
+    engine: Blink
+    engine_version: ""
+  device:
+    type: tv
+    brand: Philips
+    model: QM152E
```

Comment:
model: ""

![image](https://user-images.githubusercontent.com/1337066/152636376-76ad13af-f62d-4bbc-a97e-c0868dc3bf74.png)

QM152E - MainBoard or Version Software, used on Philips TVs for 2015-16